src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -1126,6 +1222,20 @@ public sealed partial class CertificateTrustConfigurationCallbackAnnotationConte
         public required CertificateTrustScope Scope { get { throw null; } init { } }
     }
 
+    public partial class CertificateTrustExecutionConfigurationContext
```

Comment:
@danegsta- should these 2 classes have [Experimental] on them? The only place that they are used is Experimental.

Reply:
Fixed in https://github.com/dotnet/aspire/pull/13522